dydx-testnet-4/pregenesis.json

```diff
@@ -0,0 +1,1750 @@
+          "id": 22,
+          "exponent": -9,
+          "price": 1842365656
+        },
+        {
+          "id": 23,
+          "exponent": -9,
+          "price": 6787621897
+        },
+        {
+          "id": 24,
+          "exponent": -9,
+          "price": 1127629325
+        },
+        {
+          "id": 25,
+          "exponent": -10,
+          "price": 2779565892
+        },
+        {
+          "id": 26,
+          "exponent": -9,
+          "price": 1855061997
+        },
+        {
+          "id": 27,
+          "exponent": -9,
+          "price": 1562218603
+        },
+        {
+          "id": 28,
+          "exponent": -16,
+          "price": 2481900353
+        },
+        {
+          "id": 29,
+          "exponent": -10,
+          "price": 1686998025
+        },
+        {
+          "id": 30,
+          "exponent": -15,
+          "price": 8895882688
+        },
+        {
+          "id": 31,
+          "exponent": -10,
+          "price": 5896318772
+        },
+        {
+          "id": 32,
+          "exponent": -10,
+          "price": 6327613800
+        },
+        {
+          "id": 1000000,
+          "exponent": -9,
+          "price": 1000000000
+        },
+        {
+          "id": 1000001,
+          "exponent": -9,
+          "price": 2050000000
+        }
+      ]
+    },
+    "rewards": {
+      "params": {
+        "treasury_account": "rewards_treasury",
+        "denom": "dv4tnt",
+        "denom_exponent": -18,
+        "market_id": 1000001,
+        "fee_multiplier_ppm": 0
+      }
+    },
+    "sending": {},
+    "slashing": {
+      "params": {
+        "signed_blocks_window": "12288",
+        "min_signed_per_window": "0.2",
+        "downtime_jail_duration": "3600s",
+        "slash_fraction_double_sign": "0.0",
+        "slash_fraction_downtime": "0.0"
+      },
+      "signing_infos": [],
+      "missed_blocks": []
+    },
+    "staking": {
+      "params": {
+        "unbonding_time": "2592000s",
+        "max_validators": 60,
+        "max_entries": 7,
+        "historical_entries": 10000,
+        "bond_denom": "dv4tnt",
+        "min_commission_rate": "0.05"
+      },
+      "last_total_power": "0",
+      "last_validator_powers": [],
+      "validators": [],
+      "delegations": [],
+      "unbonding_delegations": [],
+      "redelegations": [],
+      "exported": false
+    },
+    "stats": {
+      "params": {
+        "window_duration": "2592000s"
+      }
+    },
+    "subaccounts": {
+      "subaccounts": []
+    },
+    "transfer": {
+      "port_id": "transfer",
+      "denom_traces": [],
+      "params": {
+        "send_enabled": true,
+        "receive_enabled": true
+      },
+      "total_escrowed": []
+    },
+    "upgrade": {},
+    "vest": {
+      "vest_entries": [
+        {
+          "vester_account": "community_vester",
+          "treasury_account": "community_treasury",
+          "denom": "dv4tnt",
+          "start_time": "2001-01-01T00:00:00Z",
```

Comment:
@teddyding are these start + end times correct?

also, should we fund these accounts with some value so that vesting occurs?